text.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Risalamande</title>
+
+    <link rel="stylesheet" href="./style.css">
+    </head>
+<body>
+    <form id="recipe-form">
+        <h2>Add New Recipe</h2>
+        <label for="title">Recipe Title:</label>
+        <input type="text" id="title" name="title" required><br><br>
+        
+        <label for="description">Description:</label>
+        <textarea id="description" name="description" required></textarea><br><br>
+                      
+        <div id="ingredients-container">
+            <label for="ingredient1">Ingredient 1:</label>
+            <input type="text" id="ingredient1" name="ingredient1" required><br><br>
+            
+            <label for="ingredient2">Ingredient 2:</label>
+            <input type="text" id="ingredient2" name="ingredient2" required><br><br>
+            
+            <label for="ingredient3">Ingredient 3:</label>
+            <input type="text" id="ingredient3" name="ingredient3" required><br><br>
+            
+            <label for="ingredient4">Ingredient 4:</label>
+            <input type="text" id="ingredient4" name="ingredient4" required><br><br>
+            
+            <label for="ingredient5">Ingredient 5:</label>
+            <input type="text" id="ingredient5" name="ingredient5" required><br><br>
+        </div>
+
+        <button type="button" onclick="addIngredient()">Add More Ingredient</button><br><br>
```

Comment:
Prefer adding event listeners in javascript instead of putting them in the HTML.